int8 area_assigned
---
bool acceptance